### DISCUSSION TITLE
Authentication error with large SMTP mail

### DISCUSSION THREAD

**Discussion by Sven612** [postalserver/postal#2295] [Help with using Postal]
If you send a PDF file from e.g. a scanner that is larger than 10 MB, then the scanner gets an authentication error. 

Can you integrate a setting here that you can adjust the reception / transmission size.

**Comment by catphish** [comment]
Have you tried changing the max_message_size parameter?
```
smtp_server:
  max_message_size: 14
```
The default is 14MB.

**Comment by willpower232** [comment]
It would be worth watching the SMTP logs to see what is happening precisely unless the scanner will tell you what Postal told it but yeah, its likely to be the `max_message_size` thanks to network overhead etc
